### PR TITLE
Docs: Improved style, grammar and formatting.

### PR DESCRIPTION
**Pull request by hentai-chan**
I tried to improve the English translation while staying true to the original document. I also changed the formatting a little so that this README file conforms to markdown rules and recommendations.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,80 +1,92 @@
...
 
-Flask-State is a lightweight chart plugin to show machine state.
-
-* **Monitoring indicators:** CPU, Memory, Disk usage, LoadAVG, Boot time.
-* **Extensible:** It has rich options for extended functions, including redis monitoring, user authentication, custom logging, i18n and etc.
-* **Stable:** Lightweight dependencies, meanwhile solving multi-progress concurrency problems (if you use [gunicorn](https://gunicorn.org/)).
+Flask-State is a lightweight chart plugin for displaying machine state data in your web application.
 
-Flask-State is an active project, well-tested and complete update roadmap.
+* **Monitored Data:** CPU, memory, disk usage, LoadAVG and boot time.
```
Comment (FesonX) [inline README.md:13]: Maybe **Monitored Metric** is better

File: README.md [hunk 2]
```diff
@@ -1,80 +1,92 @@
...
-Flask-State follows the [Contributor Covenant](https://www.contributor-covenant.org/version/1/3/0/code-of-conduct/) Code of Conduct.
+You're welcome to [open a new issue](https://github.com/yoobool/flask-state/issues/new)!
+Flask-State follows the [Contributor Covenant](https://www.contributor-covenant.org/version/1/3/0/code-of-conduct/)
+Code of Conduct.
 
 ## Community Channel
-We're on [Gitter](https://gitter.im/flaskstate/community) ! Please join us.
+
+We're on [Gitter](https://gitter.im/flaskstate/community)! Join the conversation
+fore more questions and inquiries about this project.
```
Comment (FesonX) [inline README.md:88]: **Correct:** fore -> for more questions and inquiries about this project.

**Review by FesonX — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (hentai-chan) [reply README.md:88]: Thanks for your suggestions, I'm on it!

**Comment by hentai-chan** [comment]
I don't know Mandarin so it would be great if someone could also edit [this file](https://github.com/yoobool/flask-state/blob/v1.0.x/src/flask_state/i18n/README.zh-CN.md) to reflect the changes made in this PR. In some instances I used shorter and more concise language, take for example

```diff
- ### Secondly：Call the init_app method of the flask-state to initialize the configuration.
+ ### 2. Configure Flask-State
```

In this specific case it was obvious from the subsequent code snippet that the `init_app ` invocation is required to configure the application:

```python
import flask_state
flask_state.init_app(app)
```

**Comment by aaajiulong** [comment]
Hi @hentai-chan. Thanks for your the suggestions !

**Pull request merged**